MODULE.bazel

```diff
@@ -192,6 +194,9 @@ use_repo(
     "build_bazel_bazel_8_0_0",
 )
 
+apple_cc_configure = use_extension("@apple_support//crosstoll:setup.bzl", "apple_cc_configure_extension")
+use_repo(apple_cc_configure, "local_config_apple_cc")
+
```

Comment:
I think the plugin builds fine on macOS, or did it not work for you without this change?